wiki/de/de-Setup.md

```diff
@@ -42,15 +42,16 @@ Zuerst lass die anderen Teilnehmer wissen, wer du bist. Klick auf “Ansicht”
   <figcaption>Lass die Leute wissen, wer du bist</figcaption>
 </figure>
 
-Mindestens das Feld “Alias/Name” ausfüllen und dann das Fenster schließen.
+Fülle mindestens das Feld “Alias/Name” aus und dann kannst du dieses Fenster schließen.
 
 ## Mit einem Server verbinden und das Audio testen
 
 **Bevor du anfängst mit anderen Leuten zu musizieren, empfehlen wir, dass du dich zuerst mit einem leeren Server verbindest, um deine Konfiguration zu testen.** Und stell sicher, dass du auf das Signal vom Server hörst (wenn möglich) und nicht auf dich selbst vor Ort.
 
-Bevor du dich mit einem Server verbindest, solltest du dich selbst **nicht** hören können. Auf deinen eigenen Klang über Jamulus zu hören (und nicht den von dir selbst) ist die “Goldene Regel” und ermöglicht, dass du im Takt mit anderen spielst ([siehe FAQ](/wiki/FAQ)).
+Bevor du dich mit einem Server verbindest, solltest du dich selbst **nicht** hören können. Die “Goldene Regel” von Jamulus ist:
+Auf deinen eigenen Klang über das Programm zu hören, und nicht direkt von dir selbst. Das ermöglicht, dass du im Takt mit anderen spielst ([siehe FAQ](/wiki/FAQ)).
```

Comment:
```suggestion
auf deinen eigenen Klang über das Programm zu hören; nicht das direkte Signal von deinem Instrument. Dadurch kannst du mit anderen einfacher im Takt spielen ([siehe FAQ](/wiki/FAQ)).
```

This suggestion might be rather big and not too close to English ;-).

Reply:
Thanks, much better, I struggled with that one.